erpnext/www/book_appointment/index.css

```
@@ -45,7 +45,7 @@
 
 .time-slot.selected {
     color: white;
-    background: #5e64ff;
+    background: #2490ef;
```

```suggestion
    background: var(--primary-color);
```
This should work too?

---

Idk, I'm just using Frappe Cloud on public bench yet, so I cannot test your version, but it seems correct to me.